core/mutator.js

```diff
@@ -405,7 +405,8 @@ class Mutator extends Icon {
    */
   workspaceChanged_(e) {
     if (!(e.isUiEvent ||
-          (e.type === eventUtils.CHANGE && e.element === 'disabled') ||
+          (e.type === eventUtils.CHANGE &&
```

Comment:
The other reason to use `type` instead of `instanceof`: I have to `requireType` the appropriate event for the cast, but don't have to `require` it. That's true here and in any other place I might use `instanceof`.

Do you see any downsides to changing to `require`? It makes that type of event a required dependency that can't be shaken out, but I think that's fine for events.

Reply:
I tried changing to require and it broke compilation because there is a circular dependency that it can't break. I'm going to punt this and continue using type instead of instanceof.